frontend/server/src/Controllers/Contest.php

```diff
@@ -4876,6 +4876,42 @@ public static function apiContestants(\OmegaUp\Request $r): array {
         ];
     }
 
+    /**
+     * Removes a contest whether user is the creator
```

Comment:
```suggestion
     * Removes a contest if the user is the creator
```

"whether" se usa para decir "si X es cierto o no". aquí lo que se quiere decir es "sólamente si X es cierto" (sin el "o no" de al final).

Reply:
Listo.